src/index.html

```diff
@@ -10,13 +10,110 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo Title</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          alt="MOYO Header Image"
+        />
+      </a>
+
+      <nav class="nav-container">
+        <ul class="nav_links">
+          <li class="nav_list">
+            <a
+              href="#apple"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
You name your classes almost right, but use 2 underscores instead of 1.
Also, the `ul` tag is a tag for building a list in which each item is a `li` tag.
Replace all classes names with this in mind
```suggestion
      <nav class="nav">
        <ul class="nav_list">
          <li class="nav__item">
            <a
              href="#apple"
              class="nav__link is-active"
            >
              Apple
            </a>
          </li>
```